openpype/tools/workfiles/window.py

```diff
@@ -0,0 +1,393 @@
+import os
+import datetime
+from Qt import QtCore, QtWidgets
+
+from avalon import io
+
+from openpype import style
+from openpype.lib import (
+    get_workfile_doc,
+    create_workfile_doc,
+    save_workfile_data_to_doc,
+)
+from openpype.tools.utils.assets_widget import SingleSelectAssetsWidget
+from openpype.tools.utils.tasks_widget import TasksWidget
+
+from .files_widget import FilesWidget
+from .lib import TempPublishFiles, file_size_to_string
+
+
+class SidePanelWidget(QtWidgets.QWidget):
+    save_clicked = QtCore.Signal()
+    published_workfile_message = (
+        "<b>INFO</b>: Published workfiles you'll opened will be stored in"
+        " temp directory on your machine. Current temp size: <b>{}</b>."
```

Comment:
Message has typos `you'll opened` is invalid grammar.

```suggestion
        "<b>INFO</b>: Opened published workfiles will be stored in"
        " temp directory on your machine. Current temp size: <b>{}</b>."
```

Also, UX wise it somehow feels odd how it's in that panel there. Can't think of a better placement directly but just wanted to raise that point.